components/camel-aws-ddb/src/main/java/org/apache/camel/component/aws/ddb/DdbComponentVerifierExtension.java

```diff
@@ -46,11 +46,18 @@ public DdbComponentVerifierExtension(String scheme) {
     // Parameters validation
     // *********************************
 
+
+    /**
+     * Basic check of the parameters (they are not empty)
+     *
+     * @param parameters
+     * @return
+     */
     @Override
     protected Result verifyParameters(Map<String, Object> parameters) {
 
         ResultBuilder builder = ResultBuilder.withStatusAndScope(Result.Status.OK, Scope.PARAMETERS).error(ResultErrorHelper.requiresOption("accessKey", parameters))
-            .error(ResultErrorHelper.requiresOption("secretKey", parameters)).error(ResultErrorHelper.requiresOption("region", parameters));
+                .error(ResultErrorHelper.requiresOption("secretKey", parameters)).error(ResultErrorHelper.requiresOption("region", parameters));
```

Comment:
Please run the build with the Profile sourcecheck enabled. It will return some warnings about code style.

Reply:
The whitespaces shouldn't be there, I know, but the checkstyle does not complaint (see next comment)